cypress.dev.json

```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "http://localhost:8000",
```

Comment:
it feels like this should be the default to me. When we're in development, we're gonna wanna run against the latest changes, and the same in CI (so we don't get race conditions waiting for Netlify.) When would we want to test against the live app, if you're gonna be running a different set of Cypress tests against staging anyway?

Reply:
Yeah, that makes sense. In that case, I'll switch the defaults. I'd still want to run against the netlify at some point before deploying changes to the elm packages. I assumed that the continuous integration that runs against pull requests would be the best option, so that problems can be fixed before they get merged. 

The way I see it, the netlify app is our source of truth on what the components are that we reference in Production. Without testing against that, this whole excercise is a bit pointless. The reason that we want to test against individual components, is so that we can ignore the details on those components when we do our integration tests (especially relevant for accessibility testing, where you're checking that every element has x, y, and z required attributes). It's how we achieve separation of concerns.